pages/linux/setquota.md

```diff
@@ -0,0 +1,24 @@
+# setquota
```

Comment:
If the command it's available in more than 1 platform it should be placed on the `common/` directory. Please move the file.